playbooks/ansible-ee-network-integration-base/files/bootstrap-qradar.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I would prefer to move this file in a new playbooks/ansible-ee/qradar directory. So playbooks/ansible-ee/qradar/files/bootstrap-asa.yaml for this file in particular. Why? I would like to have all these ansible-ee related files in the same directory. This way it's easier to identify that these files are related.